### PR TITLE
Modification of the netatmo swagger client in order to handle float v…

### PR DESCRIPTION
**Pull request by clinique**
…alues of rain

Signed-off-by: Gaël L'hopital <glhopital@gmail.com>

### CODE REVIEW

**Comment by clinique** [comment]
Issue #866

File: addons/binding/org.openhab.binding.netatmo/META-INF/MANIFEST.MF [hunk 1]
```diff
@@ -14,7 +14,8 @@ Bundle-ClassPath: .,
  lib/json-20131018.jar,
  lib/commons-codec-1.8.jar,
  lib/netatmo-java-retrofit-1.0.3.jar,
```
Comment (jongj) [inline addons/binding/org.openhab.binding.netatmo/META-INF/MANIFEST.MF]: I guess this needs to go lib/netatmo-java-retrofit-1.0.3.jar ?

File: addons/binding/org.openhab.binding.netatmo/build.properties [hunk 2]
```diff
@@ -12,5 +12,6 @@ bin.includes = META-INF/,\
                lib/json-20131018.jar,\
                lib/commons-codec-1.8.jar,\
                lib/netatmo-java-retrofit-1.0.3.jar,\
-               lib/gson-2.3.1.jar
```
Comment (jongj) [inline addons/binding/org.openhab.binding.netatmo/build.properties]: same here? lib/netatmo-java-retrofit-1.0.3.jar,\ needs to be removed?

**Comment by clinique** [comment]
Thanks @jongj , I messed with the MANIFEST file apparently !

**Pull request merged**